software-engineering-policies/SecureDevelopment/SecureDevelopmentPolicy.md

```diff
@@ -141,6 +141,8 @@ The risk owner is the person who is responsible for the product. This may be the
 - The tool must be configured to run before code is released to production e.g. via a build/release pipeline.
 - The team should decide what the acceptable threshold is to prevent code from being released to production e.g. via a build/release pipeline.
 - The team must provide proof that the code is not released to production when this threshold is breached. Decisions to release even in the presence of a known vulnerability should be recorded and approved by the risk owner.
+- Where a suppression of a vulnerability has been discussed and accepted, a suitable comment must be added to the suppression file so that it can be audited by our security teams.
+- If a suppression has been added which requires a review at a later time, a `<suppress until="2024-01-01Z">` property should be added with an `until` date of one month from the initial suppression or reviewed date.
```

Comment:
This bullet is currently written in a form that is specific to the NVDChecker, other tools could be used.  Maybe this could be reworded:

```
- If a suppression has been added which requires a review at a later time, this should be expressed directly in the suppression with a next review date. For example, in the NVDChecker, a `<suppress until="2024-01-01Z">` property should be added with an `until` date of one month from the initial suppression or reviewed date.
```

Reply:
Have reworded to be more inclusive and made the dependency-checker an example rather than an implementation. Thank you for pointing this out 👍